sdk/test-utils/recorder/src/utils/index.ts

```diff
@@ -436,8 +436,39 @@ export function handleSingleQuotesInUrlPath(fixture: string): string {
   return updatedFixture;
 }
 
+/**
+ * Meant for node recordings only!
+ *
+ * Masks access tokens in the json response from nock fixtures.
+ * For example, the following part of the nock fixture/recording would be updated.
+ * from `.reply(200, {"token_type":"Bearer","expires_in":86399,"access_token":"e6z-9_g"}, [`
+ * to   `.reply(200, {"token_type":"Bearer","expires_in":86399,"access_token":"access_token"}, [`
+ *
+ * @param {string} fixture
+ */
+export function maskAccessTokenInNockFixture(fixture: string): string {
+  // Replaces only if the content-type is json
+  if (!isBrowser() && isContentTypeInNockFixture(fixture, jsonContentTypes)) {
+    // Matches the nock's reply from the fixture
+    const matches = fixture.match(/\.reply\((.*), (.*), .*/);
+    if (matches && matches[2]) {
```

Comment:
You can pass a regex to replace if you want so you can do this in one call, rather than several:

```typescript
.replace(
  /"access_token"\s*:\s*"(.+?)"/,
  '"access_token":"sometext"'
);
```

Reply:
Hmmm... that is similar to what we pass at the customizations at the tests in our SDKs.. 
![image](https://user-images.githubusercontent.com/10452642/111546853-88055780-8735-11eb-89a5-fd334c6727dc.png)

Reply:
Done!